app/javascript/vue/shared/wizard_modal.vue

```diff
@@ -1,6 +1,6 @@
 <template>
-  <div ref="modal" class="modal" tabindex="-1" role="dialog">
-    <div class="modal-dialog !w-[900px]" role="document">
+  <div ref="modal" class="modal" tabindex="-1" role="dialog" >
+    <div class="modal-dialog !w-[900px]" role="document" v-click-outside="handleOutsideClick">
```

Comment:
You don't need this hacks bootstrap had built in feature, if you want to prevent click on outside